src/init.cpp

```diff
@@ -1677,12 +1677,19 @@ bool AppInitMain(InitInterfaces& interfaces)
     }
 
     // ********************************************************* Step 11: import blocks
-
+    uint64_t bytes_needed = 0;
+    if (!chainActive.Height()) {
+        // If first startup
+        if (fPruneMode)
+            bytes_needed = nPruneTarget;
```

Comment:
Doesn't look like `nPruneTarget` is in bytes.

Reply:
`nPruneTarget` is in bytes : https://github.com/darosior/bitcoin/blob/138bf9a1d2d198410b1b15d3c285c2b12a200c3f/src/init.cpp#L1071

Reply:
So this comment should be fixed:

https://github.com/bitcoin/bitcoin/blob/56376f336548b53cf31e98a58dfb4db22cede6e5/src/validation.h#L186-L187

Reply:
Indeed.